library/psa_crypto.c

```diff
@@ -1554,9 +1577,21 @@ static psa_status_t psa_finish_key_creation(
 #if defined(MBEDTLS_PSA_CRYPTO_SE_C)
         if( driver != NULL )
         {
+            psa_se_key_data_storage_t data;
+#if defined(static_assert)
```

Comment:
We haven't started using asserts in Mbed Crypto. This is a static assert, which I've started using of my own initiative because they have no impact on code size or performance and if the compiler doesn't support them, they're simply skipped (they're always guarded by `defined(static_assert)`).